.github/workflows/test.yml

```diff
@@ -0,0 +1,88 @@
+name: KBase Blobstore test
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+    - ready_for_review
+  push:
+    # run workflow when merging to main or develop
+    branches:
+      - main
+      - master
+      - develop
+
+jobs:
+  workspace_deluxe_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - go: '1.12'
+            mongo: 'mongodb-linux-x86_64-3.6.12'
+            minio: '2019-05-23T00-29-34Z'
+            wired_tiger: 'false'
+          - go: '1.12'
+            mongo: 'mongodb-linux-x86_64-3.6.20'
+            minio: '2019-05-23T00-29-34Z'
+            wired_tiger: 'false'
+          - go: '1.12'
+            mongo: 'mongodb-linux-x86_64-3.6.20'
+            minio: '2019-05-23T00-29-34Z'
+            wired_tiger: 'true'
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: ${{matrix.go}}
+
+      - name: Install dependencies and set up test config
+        shell: bash
+        run: |
+
+          # set HOMEDIR
+          export HOMEDIR=`pwd`
+
+          # move to parent dir to install binaries etc
+          cd ..
+
+          # set up jars
+          git clone https://github.com/kbase/jars
+          export JARSDIR=`pwd`/jars/lib/jars/
```

Comment:
Can you add a TODO to switch to the auth shadow jar?

Reply:
strange, TODO is not highlighted in yml file.

Reply:
:+1: